# <p align=center>`Single Cells Datasets`</p>

This is a list of single cells datasets that are public.

Dataset name | link | nb images | type | species | modalities | other
--- | --- | --- | --- | --- | --- | --- 
BBBC (Broad Bioimage Benchmark Collection) [1] | [data](https://bbbc.broadinstitute.org/) | 11.425.066 images (52 sets) | cell and tissue | human, fly, mouse, sythetic cells, hamster, rat | fluorescent, DIC
IDR (Image Data Resource) [2] | [data](https://idr.openmicroscopy.org/) | 13.113.214 images (112 studies) | cell and tissue  | human, mouse, fly, plant and fungal cells | super-resolution 3DSIM and dSTORM, high-content chemical and siRNA screening, whole-slide histopathology imaging and live imaging of human and fungal cells and intact mice
HPA (Human Protein Atlas) [3] | [data](https://www.proteinatlas.org/) |  | cell and tissue | human | IF
LIVECell [4] | [data](https://sartorius-research.github.io/LIVECell/) [data-code](https://github.com/sartorius-research/LIVECell) | >1.6 million cells (8 cells type) | cell (from cell culture) | human, mouse | light microscopy
Cytokit [5] | [data](https://console.cloud.google.com/storage/browser/cytokit/datasets;tab=objects?prefix=&forceOnObjectsSortingFiltering=false) [code](https://github.com/hammerlab/cytokit) | | | | Multiplexed in-situ fluorescent imaging
CEM500K [6] | [data](https://www.ebi.ac.uk/empiar/EMPIAR-10592/) [code](https://github.com/volume-em/cem-dataset) | 496.544 images | | | cellular EM | in-house data + Cell Image Library, open connectome project, EMPIAR

# References

[[1](https://www.nature.com/articles/nmeth.2083)] Ljosa, V., Sokolnicki, K.L. and Carpenter, A.E., 2012. Annotated high-throughput microscopy image sets for validation. Nature methods, 9(7), pp.637-637.

[[2](https://www.nature.com/articles/nmeth.4326?report=reader)] Williams, E., Moore, J., Li, S.W., Rustici, G., Tarkowska, A., Chessel, A., Leo, S., Antal, B., Ferguson, R.K., Sarkans, U. and Brazma, A., 2017. Image Data Resource: a bioimage data integration and publication platform. Nature methods, 14(8), pp.775-781.

[[3](https://www.science.org/doi/full/10.1126/science.aal3321)] Thul, P.J., Åkesson, L., Wiking, M., Mahdessian, D., Geladaki, A., Ait Blal, H., Alm, T., Asplund, A., Björk, L., Breckels, L.M. and Bäckström, A., 2017. A subcellular map of the human proteome. Science, 356(6340), p.eaal3321.

[[4](https://www.nature.com/articles/s41592-021-01249-6#data-availability)] Edlund, C., Jackson, T.R., Khalid, N., Bevan, N., Dale, T., Dengel, A., Ahmed, S., Trygg, J. and Sjögren, R., 2021. LIVECell—A large-scale dataset for label-free live cell segmentation. Nature methods, 18(9), pp.1038-1045.

[[5](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3055-3)] Czech, E., Aksoy, B.A., Aksoy, P. and Hammerbacher, J., 2019. Cytokit: a single-cell analysis toolkit for high dimensional fluorescent microscopy imaging. BMC bioinformatics, 20(1), pp.1-13.

[[6](https://elifesciences.org/articles/65894)] Conrad, R. and Narayan, K., 2021. CEM500K, a large-scale heterogeneous unlabeled cellular electron microscopy image dataset for deep learning. Elife, 10, p.e65894.

? only human ? or also drosophila, fungi, mice, etc?

? IDR: Higher-level groupings phenotype: altered cell number, cell component number, cell cycle phase, cell cycle, cell morphology, cell population morphology, cell projection, cellular component, cyctoskeletal, nuclear, protein localization, other

? other to check: https://www.allencell.org/data-downloading.html#sectionLabelFreeTrainingData, http://www.cellimagelibrary.org/pages/datasets, 
